R/categories.R

```diff
@@ -239,3 +239,70 @@ setMethod("lapply", "Categories", function (X, FUN, ...) {
     X@.Data <- lapply(X@.Data, FUN, ...)
     return(X)
 })
+
+#' Change the id of a category for a categorical variable
+#' 
+#' @param variable the variable in a crunch dataset that will be changed
+#' @param from the (old) id identifying the category you want to change
+#' @param to the (new) id for the category
+#' @return \code{variable} with an the new category id
+#' @examples
+#' \dontrun{
+#' ds$country <- changeCategoryID(ds$country, 2, 6)
+#' ds$petloc <- changeCategoryID(ds$petloc, 2, 6)
+#' }
+#' @export 
+changeCategoryID <- function (variable, from, to) {
+    # check that variable is a category
```

Comment:
I'm all for code comments but these validation checks are self-explanatory. If you want, add a comment above all that says that you're doing input validation.